### ISSUE TITLE
Standardize loading animations

### ISSUE THREAD

**Issue by catrobson** [openshiftio/openshift.io#2870]
PARENT: #2871

We use several different loading animations across OpenShift.io.  Consider the different loading animations and standardize on when we should use what types of behaviors and visuals.

Verification criteria:
- [x] Capture all locations we have a loading behavior and what it is, post to this issue
- [x] Come up with a standard for how we handle loading behaviors for particular use cases, post to this issue and share with OSIO UXD team for review
- [x] Once standard is agreed upon, create change list for where we need to change the behaviors across OpenShift.io, create a new task with these items.
- [ ] Post the standard into the wiki Adam is building

**Comment by lwrigh** [comment]
## Loading behavior that is currently implemented

### Account home loading: 
<img width="872" alt="screen shot 2018-04-04 at 11 24 31 am" src="https://user-images.githubusercontent.com/20227374/38317387-f9bd755c-37fa-11e8-9703-2aa72719eb42.png">
When a user first logs into Openshift.io, this blue progress bar, spinner, and loading message appear while the account home page is loading. 

### Space dashboard loading: 
<img width="865" alt="space dashboard widget loading" src="https://user-images.githubusercontent.com/20227374/38318516-7c4fe4a8-37fd-11e8-8697-a31661ef1626.png">
When individual widgets in the space dashboard are loading, a spinner is featured and the header of the widget appears to be disabled.   

### Profile page loading: 
<img width="944" alt="profile page loading" src="https://user-images.githubusercontent.com/20227374/38318764-2a09d0ae-37fe-11e8-8ef9-a2751ffe4ea1.png">
When a user is waiting for the  profile page to load, empty states if the widgets featured in the profile page quickly appear before the real content is finished loading. 

### Areas loading: 
<img width="879" alt="areas loading" src="https://user-images.githubusercontent.com/20227374/38319118-077ae716-37ff-11e8-8d75-987e90fe8c89.png">
When the user is waiting for the list of their areas to load, an empty state first appears while the actual list of areas is loading. 

### Collaborators loading: 
<img width="881" alt="collaborators loading" src="https://user-images.githubusercontent.com/20227374/38319276-72eb261e-37ff-11e8-8f29-10db1e58e88f.png">
When a user is waiting for the list of collaborators within a space to load, an empty state first appears before the actual list of collaborators is finished loading.

**Comment by lwrigh** [comment]
@catrobson Here's a first pass at some suggested options for more standard loading behaviors: https://redhat.invisionapp.com/share/SJGOPZ44F83

**Comment by christianvogt** [comment]
The white 'skeleton boxes' seem out of place. Normally I see them with some kind of animated treatment which gives the user a hint that content is loading. I also feel that a loader along with those boxes is redundant. We should probably choose one or the other and then be consistent with using the boxes or spinners across the app.

Too many spinners also gives the perception of being sluggish. I've generally adopted a practice of showing spinners only after 250ms have passed (or some other appropriate time delay) if the async operation is known to return quickly and the time waiting is typically only network related. For async operations that return very quickly, the 250ms delay prevents flashing a spinner to the user. I find it better to show an empty content area for that split second vs a spinner.

**Comment by lwrigh** [comment]
@catrobson  A task list containing the changes that need to get made has been posted as an issue.
